lib/state/simperium/middleware.ts

```diff
@@ -269,6 +270,27 @@ export const initSimperium = (
             }
           }, 250);
         }
+        return result;
+      }
+
+      case 'REVISIONS_TOGGLE': {
+        const state = getState();
```

Comment:
can use `nextState` here - it's important because if we use `prevState` then `showRevisions` will be opposite of what the toggle flips it to